src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+// import java.io.;
```

Comment:
why do we need this?

Reply:
I didn't notice this line...

Reply:
The original task was in this package. That's why I left him